src/githubcity/ghcity.py

```diff
@@ -527,7 +527,9 @@ def __readAPI(self, url):
                     log_message += str(reset - now_sec)
                     log_message += " secs"
                     self.__logger.warning(log_message)
-                    time.sleep(reset - now_sec)
+                    sleep_duration = reset - now_sec
+                    if sleep_duration > 0:
```

Comment:
If `sleep_duration` is equal to 0, `if sleep_duration` would be 0. You don't need to compare with 0.